README.md

```diff
@@ -144,8 +144,15 @@ username: your-username
 1. Install [Crowdin CLI v3](https://support.crowdin.com/cli-tool/)
```

Comment:
The first 3 steps are n0t needed for the automatic upload.

Reply:
Ok, I moved them to the passage that comes after it.